src/hooks/result.inc

```diff
@@ -97,14 +56,19 @@ execution_should_success() {
 
 # Check the execution of the Jenkinsfile should succed
 # $1: Returned code by the process
-# $2: Execution log
+# $2: (Optional) Execution log
```

Comment:
Since it's Optional, you should update the `echo` command in the else block

Reply:
Changed.